### PR TITLE
fix: get the CDOClients endpoint looking like the FSBid one

### PR DESCRIPTION
**Pull request by rtirserio**
Adds relationships for employees and assignments and classifications

### CODE REVIEW

File: seeds/data/employees.json [hunk 1]
```diff
@@ -3,110 +3,134 @@
     "username": "admin",
     "ad_id": "washdc\\admin",
     "perdet_seq_num": 2,
-    "fullname": "System administrator",
+    "first_name": "System",
+    "last_name": "Administrator",
```
Comment (mjoyce91) [inline seeds/data/employees.json]: I think this change from `fullname` to `first_name`/`last_name` broke functionality elsewhere. /CDOClients is the only place I've seen the name separated into first and last. Their /Agents endpoint still only returns `fullname`. You might have to break employees out into separate models, or do some data transformation at the route level.

Comment (mjoyce91) [reply seeds/data/employees.json]: Scratch that, just needed to seed my database

File: src/services/employees.js [hunk 2]
```diff
@@ -33,25 +33,102 @@ const get_agents = async query => {
...
+        per_seq_num: emp.per_seq_num,
+        per_full_name: emp.fullname,
+        per_first_name: emp.first_name,
+        per_last_name: emp.last_name,
+        per_middle_name: emp.middle_name,
+        per_prefix_name: emp.prefix_name,
+        per_birth_date: emp.dob,
+        per_org_code: "",
+        per_skill_code: skill1.skl_code,
+        per_skill_2_code: skill2.skl_code,
```
Comment (mjoyce91) [inline src/services/employees.js]: I think the answer to https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/291/files#r374315359 is here. You should also be providing `per_skill_code_desc`, `per_skill_2_code_desc`, etc

File: src/services/employees.js [hunk 3]
```diff
@@ -33,25 +33,102 @@ const get_agents = async query => {
...
-      skill_code_desc: skill1.skill_descr,
-      skill2_code: skill2.skl_code,
-      skill2_code_desc: skill2.skill_descr,
-      skill3_code: skill3.skl_code,
-      skill3_code_desc: skill3.skill_descr,
-      emplid: emp.username,
-      role_code: roles.map(r => r.code),
-      pos_location_code: location.code,
-      hs_cd: emp.hs_cd
+      rl_cd: roles.map(r => r.code),
```
Comment (mjoyce91) [inline src/services/employees.js]: I'm only seeing this as a string in the /CDOClients response

File: src/services/employees.js [hunk 4]
```diff
@@ -33,25 +33,102 @@ const get_agents = async query => {
...
+      employee: {
+        per_seq_num: emp.per_seq_num,
+        per_full_name: emp.fullname,
+        per_first_name: emp.first_name,
+        per_last_name: emp.last_name,
+        per_middle_name: emp.middle_name,
+        per_prefix_name: emp.prefix_name,
+        per_birth_date: emp.dob,
```
Comment (mjoyce91) [inline src/services/employees.js]: This comes through on /Persons but not /Clients

File: src/services/employees.js (hunk 3 above)
Comment (rtirserio) [reply src/services/employees.js]: Updated to use the first role they have

File: src/services/employees.js (hunk 2 above)
Comment (rtirserio) [reply src/services/employees.js]: this is fixed

**Comment by rtirserio** [comment]
@mjoyce91 updates made

File: src/services/employees.js [hunk 5]
```diff
@@ -34,24 +34,52 @@ const get_agents = async query => {
...
+        per_middle_name: emp.middle_name,
+        per_skill_code: skill1.skl_code,
+        per_skill_code_desc: skill1.skill_descr,
+        per_skill_2_code: skill2.skl_code,
+        per_skill_2_code_desc: skill2.skill_descr,
+        per_pay_plan_code: "",
+        per_tenure_code: "",
+        currentAssignment: {
+          ags_seq_num: currentassignment.ags_seq_num,
+          pos_seq_num: position.pos_seq_num,
```
Comment (mjoyce91) [inline src/services/employees.js]: `pos_seq_num` comes through as string on FSBid API

File: src/services/employees.js [hunk 6]
```diff
@@ -34,24 +34,52 @@ const get_agents = async query => {
...
-      skill2_code_desc: skill2.skill_descr,
-      skill3_code: skill3.skl_code,
-      skill3_code_desc: skill3.skill_descr,
-      emplid: emp.username,
-      role_code: roles.map(r => r.code),
-      pos_location_code: location.code,
-      hs_cd: emp.hs_cd
+      rl_cd: roles.length > 0 ? roles[0]['code'] : '', // FSBid only returns one role
+      employee: {
+        pert_external_id: emp.per_seq_num,
```
Comment (mjoyce91) [inline src/services/employees.js]: `pert_external_id` comes through as string on FSBid API

**Pull request merged**